test/Bridges/Variable/flip_sign.jl

```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
         @test MOI.get(bridged_mock, MOI.VariablePrimalStart(), y) == 1
     end
 
+    @testset "Test Mock model" begin
```

Comment:
Put this testset inside the `lin2v` testset.

Reply:
Only this or both testsets inside in line2v?